docs/references/react/add-react-router.mdx

```diff
@@ -33,6 +33,10 @@ description: Learn how to add React Router to your React application using their
 
 # Add React Router to your Clerk-powered React application
 
+<Callout type="warning">
+  [Loaders](https://reactrouter.com/en/main/route/loader) and [Actions](https://reactrouter.com/en/main/route/action) are currently not supported by Clerk at the moment with React Router.
+</Callout>
+
```

Comment:
The callout content is good. However please update this with the callout markdown from the CONTRIBUTING guide.